lib/openstudio-standards/utilities/object_info.rb

```diff
@@ -0,0 +1,18 @@
+# Get hash of object field names and values
+# author: Xuechen (Jerry) Lei
+# param: obj [object]
+# return: [Hash<String>] FieldName:Value
```

Comment:
Are these YARD compatible comments? We usually use `@param`, `@author`, and `@return`.

Reply:
modified